src/System.Devices.Gpio/System.Devices.Gpio/UnixDriver.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.IO.MemoryMappedFiles;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Devices.Gpio
+{
+    public class UnixDriver : GpioDriver
+    {
+        private const string GpioPath = "/sys/class/gpio";
+
+        private ISet<int> ExportedPins;
```

Comment:
private fields should start with `_` and follow camel casing:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md